### PR TITLE
docker: Updated dockerfile

### PR DESCRIPTION
**Pull request by chaitanya1731**
Added information about Intel Data Center GPU Driver components in the description label.

Signed-off-by: chaitanya1731 <chaitanya.kulkarni@intel.com>

### CODE REVIEW

**Comment by chaitanya1731** [comment]
The labels would look something like this - 
```
"Labels": {
     "architecture": "x86_64",
     "build-date": "2023-02-07T16:17:52",
     "com.redhat.component": "ubi8-minimal-container",
     "com.redhat.license_terms": "https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI",
     "description": "Intel® Data Center GPU Driver container image designed for Red Hat OpenShift Container Platform. The driver container is based on Intel Data Center GPU driver components - PMT:23WW06.5_555_MAIN, i915:RHEL86_23WW6.5_555_6469.0.3_221221.3, and Firmware:23WW06.5_555 and is tested on OCP kernel version 4.18.0-372.40.1.el8_6.x86_64.",
     "distribution-scope": "public",
     "io.buildah.version": "1.27.3",
     "io.k8s.description": "The Universal Base Image Minimal is a stripped down image that uses microdnf as a package manager. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
     "io.k8s.display-name": "Red Hat Universal Base Image 8 Minimal",
     "io.openshift.expose-services": "",
     "io.openshift.tags": "minimal rhel8",
     "maintainer": "Red Hat, Inc.",
     "name": "intel-data-center-gpu-driver-container",
     "release": "4.18.0-372.40.1.el8_6.x86_64",
     "summary": "Intel® Data Center GPU Driver Container Image",
     "url": "https://access.redhat.com/containers/#/registry.access.redhat.com/ubi8-minimal/images/8.7-1049.1675784874",
     "vcs-ref": "146fdafc2595e26f5f9c1b9a2b3f36bbca8237e4",
     "vcs-type": "git",
     "vendor": "Intel®",
     "version": "0.1.0"
},
```

File: docker/intel-dgpu-driver.Dockerfile [hunk 1]
```diff
@@ -45,13 +45,19 @@ RUN git clone -b ${FIRMWARE_RELEASE} --single-branch https://github.com/intel-gp
...
-LABEL name='intel-data-center-gpu-driver-container'
-LABEL summary='Intel® Data Center GPU Driver Container Image'
-LABEL description='Intel® Data Center GPU Driver container image for Red Hat OpenShift Container Platform'
+LABEL vendor="Intel®"
+LABEL version="${DRIVER_VERSION}"
+LABEL release="${KERNEL_FULL_VERSION}"
+LABEL name="intel-data-center-gpu-driver-container"
+LABEL summary="Intel® Data Center GPU Driver Container Image"
+LABEL description="Intel® Data Center GPU Driver container image designed for Red Hat OpenShift Container Platform. \
+The driver container is based on Intel Data Center GPU driver components - PMT:${PMT_RELEASE}, i915:${I915_RELEASE}, and Firmware:${FIRMWARE_RELEASE} \
```
Comment (uMartinXu) [inline docker/intel-dgpu-driver.Dockerfile]: Can we change to the line into

PMT driver release: ${PMT_RELEASE}, i915 driver release: ${I915_RELEASE}, and Firmware release: ${FIRMWARE_RELEASE}

File: docker/intel-dgpu-driver.Dockerfile [hunk 2]
```diff
@@ -45,13 +45,19 @@ RUN git clone -b ${FIRMWARE_RELEASE} --single-branch https://github.com/intel-gp
...
+and is tested on OCP kernel version ${KERNEL_FULL_VERSION}."
```
Comment (uMartinXu) [inline docker/intel-dgpu-driver.Dockerfile]: Can we change the line into:
This driver container image is for RHOCP-4.12 RHCOS Kernel version ${KERNEL_FULL_VERSION}"

Comment (chaitanya1731) [reply docker/intel-dgpu-driver.Dockerfile]: done

File: docker/intel-dgpu-driver.Dockerfile (hunk 1 above)
Comment (chaitanya1731) [reply docker/intel-dgpu-driver.Dockerfile]: done

**Comment by hershpa** [comment]
LGTM, we are now leveraging the build args in the 2nd stage.

**Comment by hershpa** [comment]
> Added information about Intel Data Center GPU Driver components in the summary label.
> 
> Signed-off-by: chaitanya1731 [chaitanya.kulkarni@intel.com](mailto:chaitanya.kulkarni@intel.com)

We are updating the description label right?

File: docker/intel-dgpu-driver.Dockerfile [hunk 3]
```diff
@@ -45,13 +45,19 @@ RUN git clone -b ${FIRMWARE_RELEASE} --single-branch https://github.com/intel-gp
...
-LABEL summary='Intel® Data Center GPU Driver Container Image'
-LABEL description='Intel® Data Center GPU Driver container image for Red Hat OpenShift Container Platform'
+LABEL vendor="Intel®"
+LABEL version="${DRIVER_VERSION}"
+LABEL release="${KERNEL_FULL_VERSION}"
+LABEL name="intel-data-center-gpu-driver-container"
+LABEL summary="Intel® Data Center GPU Driver Container Image"
+LABEL description="Intel® Data Center GPU Driver container image designed for Red Hat OpenShift Container Platform. \
+The driver container is based on Intel Data Center GPU driver components - PMT driver release:${PMT_RELEASE}, i915 driver release:${I915_RELEASE}, \
+and Firmware driver release:${FIRMWARE_RELEASE}. This driver container image is supported for RHOCP 4.12 RHCOS kernel version: ${KERNEL_FULL_VERSION}."
```
Comment (uMartinXu) [inline docker/intel-dgpu-driver.Dockerfile]: It is Firmware, not Firmware driver.

Comment (chaitanya1731) [reply docker/intel-dgpu-driver.Dockerfile]: Done.

**Comment by uMartinXu** [comment]
the PR looks good to me. @hershpa what are your opinions?

**Comment by hershpa** [comment]
Looks good. My only minor feedback is we should update the commit message to indicate we are updating the description label. Currently, we state we are updating the summary label.

**Comment by chaitanya1731** [comment]
Nice catch @hershpa I amended the commit message.

**Comment by hershpa** [comment]
Cool

**Comment by uMartinXu** [comment]
the PR is good to merge. Thanks!

**Pull request merged**